.github/workflows/update_zds_android.yml

```diff
@@ -0,0 +1,19 @@
+name: Update zds_android
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      PAT:
+        required: true
+
+jobs:
+  publish_android:
+    uses: ./.github/workflows/copy-content.yml
+    secrets: inherit
+    with:
+      repo: ZebraDevs/zds_android
+      branch: "update-zeta-icons"
+      source_dir: "./outputs/android/."
+      destination_dir: "app/src/main/res/drawable"
```

Comment:
I think this destination is wrong. Should be:
` components/src/main/res/drawable`

Reply:
are you sure? the drawable directory in app is where the icons are currently stored

Reply:
that is the example app, that does not get distributed. 
I guess when this repo was made it was not expected to share those icons